app/javascript/components/enrollment/steps/Exposure.js

```diff
@@ -432,15 +432,15 @@ class Exposure extends React.Component {
             controlId="first_positive_lab"
             className="mb-2">
             {this.state.current.first_positive_lab && (
-              <div className={`mx-1 mb-2 {this.state.current.patient.no_symptom_history ? '' : 'disabled-first-positive-lab'}`}>
+              <div className={`mx-1 mb-2 {this.state.current.patient.no_reported_symptoms ? '' : 'disabled-first-positive-lab'}`}>
                 <div className="section-header">
                   <Form.Label className="nav-input-label">FIRST POSITIVE LAB RESULT</Form.Label>
```

Comment:
This was created as a `Form.Label` component for the styling. What you said makes sense, but I think it's something to be done in the future given that this is a hotfix into master which is to be tagged soon because it would require styling the div and making sure everything is still displayed the way it was before

Reply:
I will let @holmesie be the decider on this because it is a 508 compliance issue

Reply:
This needs to be addressed; 508 compliance issues are not acceptable. Understood this wasn't introduced with this change specific PR but I'd prefer it be addressed in line with the label change. Don't worry about timing; goal is to tag the final release Monday @timwongj. @sgober can you help Tim replace this with the correct element?

Reply:
done: https://github.com/SaraAlert/SaraAlert/pull/814/commits/12b233b4d63fc917c3e6f903e9370160a9365205

Reply:
Can we make this a div and change the class applied to a new class that contains the following styling?  Perhaps in the new `enrollment.scss` file?

```
    display: inline;
    text-transform: uppercase;
    font-weight: 600;
```

This will make the styling the same as the other headers